DOCKER/docker-compose

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+
+services:
+
+  postgis:
+    build:
+      context: .
+      dockerfile: dockerfile-toms-test-postgis
+    hostname: toms_postgis2
+    environment:
+      - POSTGRES_PASSWORD=postgis
+      - POSTGRES_DBNAME=gis
+    volumes:
+      - "pg_data_toms_test:/var/lib/postgresql"
+      - /home/QGIS/plugins/TOMs/DATAMODEL:/docker-entrypoint-initdb.d
+      - /home/QGIS/plugins/TOMs/test/data:/docker-entrypoint-initdb.d
+    ports:
+      - 5435:5432
+
+  qgisserver:
+    image: openquake/qgis-server:ltr
+    hostname: qgis-server1
+    environment:
+      - PGSERVICEFILE=/io/pg_service/.pg_service.conf
```

Comment:
Does this file exist in the container ?
It looks like you need to create the file and mount it in the volumes section below, just like you're doing for other files

Reply:
You are correct. It should be present now.